src/main/java/reposense/parser/GroupConfigCsvParser.java

```diff
@@ -79,4 +84,12 @@ private static GroupConfiguration findMatchingGroupConfiguration(
         results.add(config);
         return config;
     }
+
+    /**
+     * Gets the number of columns expected in the header.
```

Comment:
Be more specific in which header are we referring to here (in this case the group config header). Same for the other two

Reply:
Thanks for the feedback. I removed the getHeader method entirely, since we are passing the expected header size to the constructor of CsvParser